samples/ChatSample/ChatSample/appsettings.json

```diff
@@ -2,7 +2,17 @@
   "Azure": {
     "SignalR": {
       "Enabled": true,
-      "ConnectionString": ""
+      "ConnectionString": "",
+      "DiagnosticLogs": [
```

Comment:
I am thinking if there is any way to leverage the `Logging` settings https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-3.1

Reply:
But this option is about tracking client, and it's only an setting for Azure SignalR. Looks like it's fine to place in `Azure:SignalR` section. Maybe the improve the naming like this:
```
  "Azure": {
    "SignalR": {
      "Enabled": true,
      "ConnectionString": "",
      "TrackingClient": [
        {
          "LogType": "Messaging",
          "Accept": true
        },
        {
          "LogType": "Connectivity",
          "Accept": false
        }
      ]
    }
  }
```